### PR TITLE
[bitnami/parse] feat: :lock: Enable networkPolicy

### PR DESCRIPTION
**Pull request by javsalgar**
Signed-off-by: Javier Salmeron Garcia <jsalmeron@vmware.com>

### Description of the change

This PR normalizes the use of NetworkPolicy in the chart. Adds all Bitnami standards for NetworkPolicies as well as enabling it by default, in order to comply with security checklists.

### Benefits

More security in the chart

### Applicable issues

- fixes #

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/parse/templates/server-networkpolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  egress:
+    # Allow dns resolution
+    - ports:
+        - port: 53
+          protocol: UDP
+        - port: 53
+          protocol: TCP
+    # Allow access to mongodb
+    - ports:
+        {{/* TODO: Allow changing the mongodb port in the chart YAML */}}
```
Comment (FraPazGal) [inline bitnami/parse/templates/server-networkpolicy.yaml:34]: Is this to be done as part of the PR or in a future one?

File: bitnami/parse/templates/dashboard-networkpolicy.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+          podSelector:
+            matchLabels:
+              {{- range $key, $value := .Values.dashboard.networkPolicy.ingressNSPodMatchLabels }}
+              {{ $key | quote }}: {{ $value | quote }}
+              {{- end }}
+          {{- end }}
+        {{- end }}
+        - podSelector:
+            matchLabels: {{- include "common.labels.matchLabels" ( dict "customLabels" $podLabels "context" $ ) | nindent 14 }}
+              app.kubernetes.io/part-of: vault
```
Comment (FraPazGal) [inline bitnami/parse/templates/dashboard-networkpolicy.yaml:68]: This looks like a typo. Same as below in `server-networkpolicy.yaml`

**Review by FraPazGal — COMMENTED** [review]
@javsalgar I added a couple of comments as a review, please check them whenever you can.

File: bitnami/parse/templates/server-networkpolicy.yaml (hunk 1 above)
Comment (javsalgar) [reply bitnami/parse/templates/server-networkpolicy.yaml:34]: Future

**Review by FraPazGal — DISMISSED** [review]
LGTM!

**Review by FraPazGal — APPROVED** [review]

**Pull request merged**